web/src/components/repo/RepoItems.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <router-link v-if="repo" :to="{ name: 'repo', params: { repoId: repo.id } }">
+    <div
+      class="flex flex-col border rounded-md bg-wp-background-100 overflow-hidden p-4 border-wp-background-400 dark:bg-wp-background-200 cursor-pointer hover:shadow-md hover:bg-wp-background-300 dark:hover:bg-wp-background-300"
+    >
+      <div class="grid grid-cols-[1.5rem,auto,1fr] gap-2 items-center">
+        <PipelineStatusIcon v-if="pipeline" :status="pipeline.status" />
+        <Icon v-else name="new" class="text-wp-text-100" />
+        <span class="text-wp-text-100 text-lg">{{ `${repo.owner} / ${repo.name}` }}</span>
+        <span class="ml-auto">
+          <Badge
+            v-if="repo.visibility === RepoVisibility.Public"
+            :label="$t('repo.settings.general.visibility.public.public')"
```

Comment:
Could you bring those translations a bit to the top level so sth like: `visibility.public`.

Reply:
Not sure what you mean. I have just used the existing translation from `en.json` and not added a new one.

Reply:
Yep. You are using `repo.settings.general.visibility.public.public` which was for the settings page as you can see in the name. As duplicating such entries makes no sense IMO, I started to move things like `repo.settings.general.visibility.public.public` to a rather flat property like `visibility.public` away from the place it's used to rather match just the content.

Reply:
Ah ok got it :+1: will take care

Reply:
Done